lib/Models/Leaflet.ts

```diff
@@ -104,6 +84,18 @@ export default class Leaflet extends GlobeOrMap {
   private _disposeSelectedFeatureSubscription?: () => void;
   private _disposeSplitterReaction: () => void;
 
+  // These are used to split CesiumTileLayer and MapboxCanvasVectorTileLayer
+  @observable size: L.Point | undefined;
+  @observable nw: L.Point | undefined;
+  @observable se: L.Point | undefined;
+
+  @action
+  private updateMapObservables() {
+    this.size = this.map.getSize();
+    this.nw = this.map.containerPointToLayerPoint([0, 0]);
+    this.se = this.map.containerPointToLayerPoint(this.size);
+  }
+
```

Comment:
These used to be in `CesiumTileLayer`, but are now here so the splitter works properly in Leaflet while map is panning/zooming